include/deal.II/base/parameter_handler.h

```diff
@@ -1158,7 +1158,8 @@ class ParameterHandler : public Subscriptor
    */
   void
   add_action(const std::string &                                  entry,
-             const std::function<void(const std::string &value)> &action);
+             const std::function<void(const std::string &value)> &action,
+             const bool execute_action = true);
```

Comment:
You'll have to adjust the documentation of the function accordingly. The first bullet point of the documentation states that the action is executed at the end of the call to this function, but this is now no longer generally true.